utils/parametric/spec/otel_trace.py

```diff
@@ -1,17 +1,7 @@
 from typing import TypedDict
+from enum import Enum
```

Comment:
In test_parametric_endpoints, I document usages of the `SpanKind` and `StatusCode` enums. I think this is a better interface than asking developers to use a set of constants.